extensions/Dockerfile

```diff
@@ -32,3 +32,13 @@ RUN createrepo_c /usr/share/rpm-ostree/extensions/
 ## Final container that has the extensions repo dir
 FROM registry.access.redhat.com/ubi9/ubi:latest
 COPY --from=builder /usr/share/rpm-ostree/extensions/ /usr/share/rpm-ostree/extensions/
+
+# Generate extensions.json for meta.json, written to a bind-mounted path during the build.
+# Use dnf repoquery to print 'name: version,' for each RPM
+# sed to remove the comma from the last RPM
+RUN sh -c 'echo "{" > /tmp/extensions.json && \
```

Comment:
Hmm, pretty sure even though this doesn't create anything in the rootfs, this is still going to add a layer to the final image. Instead, we can run this from the `builder` stage right after creating the repo.

Reply:
SGTM